src/conversions/chrono.rs

```diff
@@ -729,6 +730,30 @@ mod tests {
         })
     }
 
+    #[test]
+    fn test_pyo3_offset_fixed_frompyobject_created_in_python() {
+        Python::with_gil(|py| {
+            let code = r#"
+import datetime
+t = datetime.datetime.fromtimestamp(12345).replace(tzinfo=datetime.timezone(datetime.timedelta(hours=-8)))
```

Comment:
Since the particular value of the timestamp should not matter, this looks like a good candidate for a proptest by interpolate the timestamp and the offset into the Python code?

Reply:
Wouldn't it make sense to make the timezone offset variable for the proptest as well?